src/main/java/com/thealgorithms/maths/StrobogrammaticNumber.java

```diff
@@ -0,0 +1,48 @@
+package com.thealgorithms.maths;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StrobogrammaticNumber {
+
+    public static void main(String[] args) {
+        assert isStrobogrammatic("69") == true;
+        assert isStrobogrammatic("88") == true;
+        assert isStrobogrammatic("818") == true;
+        assert isStrobogrammatic("101") == true;
+        assert isStrobogrammatic("609") == false;
+        assert isStrobogrammatic("120") == false;
+    }
```

Comment:
Instead of `main()` method write some Junit test. You can refer to https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms/maths